hathor/consensus/consensus_settings.py

```diff
@@ -112,5 +130,12 @@ def _get_valid_vertex_versions(self, include_genesis: bool) -> set[TxVersion]:
 
         return versions
 
+    @override
+    def get_peer_hello_hash(self) -> str | None:
```

Comment:
This is immutable. Should we calculate it once and just cache the digest?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1078/commits/b58b125dae7da71d39f750fa53d8729f4efe3463

Reply:
Thanks!